docs/install/install-server/install-server.inc

```diff
@@ -59,36 +59,102 @@ Install the software
         The first package (inmanta-oss) contains all the code and the commands. The server and the agent packages install config
         files and systemd unit files. The dashboard is installed with the server package.
 
-    .. tab:: Other Linux and Mac
+
+    .. tab:: Debian, Ubuntu and derivatives. 
 
         First make sure Python >= 3.6 and git are installed. Inmanta requires many dependencies so it is recommended to create a virtual env.
         Next install inmanta with pip install in the newly created virtual env.
 
+        Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
+
         .. code-block:: sh
 
-            # Install python3 >= 3.6 and git
+            # Install GCC, Python pip and Wheel
+            sudo apt-get update
+            sudo apt-get install build-essential
+            sudo apt-get install python3-pip
+
+            pip3 install wheel
+            # OR
+            pip install wheel
```

Comment:
Wheel should be installed into the `/opt/inmanta` venv instead of the global environment.

Reply:
done